docs/main/external-coupling.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Coupling external models
+---
+
+Complementary external models can be coupled to the ETM, with the aim to generate insights that might not be achieved with each model on its own. For example, the ETM has no explicit geographical allocation of technical components. The ETM could therefore be coupled to a model that is strongly geographically oriented, in order to evaluate the spatial effects of the system interactions from the ETM.
+
+## Implementation
+
+### Coupling external inputs
+One way to couple an external model is to implementing specific outcomes from the model in the ETM. This can be done by using dedicated external coupling inputs, that bypass the sliders presented in the ETM front-end.
```

Comment:
```suggestion
One way to couple an external model is to implement specific outcomes from the model in the ETM. This can be done by using dedicated external coupling inputs, that bypass the sliders presented in the ETM front-end.
```